projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -15,6 +15,13 @@
  * Class WPcom_Admin_Menu.
  */
 class WPcom_Admin_Menu extends Admin_Menu {
+	/**
+	 * Holds the current plan, set by get_current_plan().
+	 *
+	 * @var ?array
+	 */
+	private $current_plan = null;
```

Comment:
I personally prefer to keep method return types and property types as singular and stable as possible. I saw it being set to an empty array in a previous iteration of this patch, which felt cleaner to me. Just a note for consideration, don't feel too strongly about it.

Reply:
Sounds good to me! 👍